src/uct/api/uct.h

```diff
@@ -1340,14 +1340,7 @@ ucs_status_t uct_ep_connect_to_ep(uct_ep_h ep, const uct_device_addr_t *dev_addr
  * @param [in]  length           Length of the private data.
  * @param [out] ep_p             Handle to the created endpoint.
  *
- * @return UCS_OK              - The connection to the server, on the client side,
- *                               was established. No reply from the server is
- *                               required and therefore the reply_cb callback
- *                               won't be invoked.
- * @return UCS_INPROGRESS      - The connection to the remote peer was initiated.
- *                               The user will be notified of the connection
- *                               establishment, on the client side, when the reply_cb
- *                               callback will be invoked.
+ * @return UCS_OK              - Connection was request was sent to the server.
```

Comment:
I think this should be something like:
  Connection was successfully established with the server.

Reply:
in this case, it doesn't guarantee that the connection was established - it merely sends the request. (similar to am_bcopy, in a way). the server has to extract addresses from private_data and send a reply.
if there is a failure, it would be reported through error handling mechanism
(now i guess this comment better be added to the doxygen, right?)

Reply:
Thanks for the clarification. Yes, please update the doxygen accordingly :)